plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -400,6 +402,25 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
         export(res.getOutputStream());
     }
 
+    /**
+     * Export JSONSchema to URL
+     * @throws Exception
+     */
+    @RequirePOST
+    public void doExportJSONSchema(StaplerRequest req, StaplerResponse res) throws Exception {
+
+        if (!Jenkins.getInstance().hasPermission(Jenkins.ADMINISTER)) {
+            res.sendError(HttpServletResponse.SC_FORBIDDEN);
+            return;
+        }
+        writeJSONSchema();
+        res.setContentType("application/x-yaml; charset=utf-8");
+        res.addHeader("Content-Disposition", "attachment; filename=JSONSchema.json");
```

Comment:
I don't think we should offer it as an attachment.
```suggestion
```

Reply:
How do we export then :P